File: app/src/main/java/chat/rocket/android/helper/Logger.java
package chat.rocket.android.helper;

import com.crashlytics.android.Crashlytics;
import com.google.firebase.crash.FirebaseCrash;

public class Logger {

    public static void report(Throwable throwable) {
        FirebaseCrash.report(throwable);
        Crashlytics.logException(throwable);
    }
}

File: app/src/main/java/chat/rocket/android/helper/Logger.kt
package chat.rocket.android.helper

import chat.rocket.android.BuildConfig
import com.crashlytics.android.Crashlytics
import com.google.firebase.crash.FirebaseCrash

object Logger {

    fun report(throwable: Throwable) {
        if (BuildConfig.DEBUG) {
            throwable.printStackTrace()
        }
        FirebaseCrash.report(throwable)
        Crashlytics.logException(throwable)
    }
}
